routes/api.php

```diff
@@ -86,6 +86,10 @@
         Route::get('rooms/{room}/files/{file}', 'RoomFileController@show')->name('rooms.files.show')->middleware(['can:downloadFile,room,file', 'room.authenticate']);
     });
 
+
+    Route::get('roomTypes', 'RoomTypeController@index')->name('roomTypes.index');
```

Comment:
Should the room types be listable even for unauthorized users?

Reply:
This is nothing that wouldn't need special protection, but to make it future proof (some room types only for some groups) I'll move it to the middleware